tests/Feature/Mship/NotificationTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+    namespace Tests\Feature\Mship;
```

Comment:
Not sure why StyleCI isn't quite picking it up but there is a fair amount of indentation on this file and the Factory?